pkgs/development/compilers/ghc/8.4.2.nix

```diff
@@ -24,7 +24,15 @@
   # platform). Static libs are always built.
   enableShared ? !targetPlatform.useAndroidPrebuilt
 
+, # Whetherto build terminfo.
+  enableTerminfo ? !targetPlatform.isWindows
+
 , version ? "8.4.2"
+
+  # What flavour to build. An empty string indicates no
+  # specific flavour and falls back to ghc default values.
+, ghcFlavour ? ""
```

Comment:
`, ghcFlavour = stdenv.lib.optionalString (targetPlatform != hostPlatform) "perf-cross" `

Reply:
Can we also backport this change to the earlier GHCs for consistency?

Reply:
We can. But I'd argue against it. I'd rather see us align the 8.{4,6}.y and head into a `common` form, and then graft previous version onto the `common.nix`. Otherwise we are going to end up backporting this stuff every time again across all expressions.